src/model.py

```diff
@@ -220,6 +228,23 @@ def create_response(self, vllm_output):
         )
         return pb_utils.InferenceResponse(output_tensors=[triton_output_tensor])
 
+    def create_stream_response(self, vllm_output, last_outputs):
+        """
+        Parses the output from the vLLM engine, extracts only newly generated
+        text and packs it into Triton response.
+        """
+        if last_outputs is None:
```

Comment:
```suggestion
        if previous_outputs is None:
```
Small suggestion here, in another context I was reminded that "last" can also be confused with "final", "previous" makes a little clearer - but just a suggestion.